ihatemoney/web.py

```diff
@@ -534,11 +536,23 @@ def export_project(file, format):
     )
 
 
-@main.route("/exit")
+@main.route("/exit", methods=["GET", "POST"])
 def exit():
-    # delete the session
-    session.clear()
-    return redirect(url_for(".home"))
+    # We must test it manually, because otherwise, it creates a project "exit"
+    if request.method == "GET":
+        abort(405)
```

Comment:
Why not just remove GET from the supported methods for this route?

Reply:
I tried this, but then, entering `/exit` will redirect to the project creation page with the project id `exit`. I bet this is unexpected, and prefer to cover this weird use case with custom handling.

Reply:
Ah, annoying indeed...